sklearn-demo.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>I think this is wrong. Should be 0.815</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maxjakob/reviewnb-playground/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>ability to add emojis is dearly missed here</p>